### PR TITLE
First steps on the new redesign.

### PR DESCRIPTION
**Pull request by pablortsal**
New components:
- creator-badge
- social-button
- community-item

I added some new css code, to fit the new design.
Its a lot of changes so please feel free to decline if anything need to be fix =)

### CODE REVIEW

**Comment by pablortsal** [comment]
Hey guys I Updated the pull request, since @Caballerog told me that i had some CSS bugs, so I think the problem is solved right now =)

**Comment by Caballerog** [comment]
I have made some changes to the PR of @pablortsal , for my part it is approved.

Waiting for @tonivj5  review.

We continue working colleagues!

**Comment by tonivj5** [comment]
Good work! I'll review it today :smile_cat:

File: src/app/pages/main/components/community-preview/community-preview.component.html [hunk 1]
```diff
@@ -1,41 +1,12 @@
...
-      >
-        <picture>
-          <source
-            *ngFor="let media of responsiveImagesConfiguration"
-            [media]="'(max-width: ' + media.max + 'px) and (min-width: ' + media.min + 'px)'"
-            [srcset]="'./assets/images/' + community.id + media.suffix + '.' + media.format"
-            [type]="'image/' + media.format"
-          />
+      <ngcommunity-community-item *cdkVirtualFor="let community of filtered$ | async" [community]="community"
+        (communitySelected)="onSelected(community.name)"></ngcommunity-community-item>
```
Comment (tonivj5) [inline src/app/pages/main/components/community-preview/community-preview.component.html:8]: I'd use native `click` event

```suggestion
        (click)="onSelected(community.name)"></ngcommunity-community-item>
```

File: src/app/shared/components/community-item/community-item.component.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+@Component({
+  selector: 'ngcommunity-community-item',
+  templateUrl: './community-item.component.html',
+  styleUrls: ['./community-item.component.scss'],
+})
+export class CommunityItemComponent {
+  @Input() community: Community;
+  @Input() detailCommunity = false;
+  @Output() communitySelected: EventEmitter<void> = new EventEmitter();
```
Comment (tonivj5) [inline src/app/shared/components/community-item/community-item.component.ts:13]: This event emiter could be removed

File: src/app/shared/components/community-item/community-item.component.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    {
+      min: 1920,
+      max: 5000,
+      suffix: '_xl',
+      format: 'png',
+    },
+  ];
+  constructor() {}
+
+  onCommunitySelected() {
```
Comment (tonivj5) [inline src/app/shared/components/community-item/community-item.component.ts:79]: And this piece of code could be removed

File: src/app/shared/components/community-item/community-item.component.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    },
+
+    {
+      min: 1920,
+      max: 5000,
+      suffix: '_xl',
+      format: 'png',
+    },
+  ];
+  constructor() {}
```
Comment (tonivj5) [inline src/app/shared/components/community-item/community-item.component.ts:77]: remove empty constructor

File: src/app/pages/main/components/community-preview/community-preview.component.html (hunk 1 above)
Comment (Caballerog) [reply src/app/pages/main/components/community-preview/community-preview.component.html:8]: Good catch!

**Comment by Caballerog** [comment]
This PR is ready to be merge!

**Pull request merged**